python_modules/dagster/dagster_tests/core_tests/instance_tests/test_instance_config.py

```diff
@@ -48,8 +49,81 @@ def test_concurrency_config(config_filename, caplog):
         "error_run_queue_concurrency_mismatch.yaml",
     ),
 )
-def test_concurrency_config_mismatch(config_filename, caplog):
+def test_concurrency_config_mismatch(config_filename):
     base_dir = file_relative_path(__file__, "./test_config")
     with environ({"DAGSTER_HOME": base_dir}):
         with pytest.raises(DagsterInvalidConfigError, match="the `concurrency > "):
             dagster_instance_config(base_dir, config_filename)
+
+
+def test_legacy_concurrency_enabled_run_blocking():
+    base_dir = file_relative_path(__file__, "./test_config")
+    with environ({"DAGSTER_HOME": base_dir}):
+        instance_config, _ = dagster_instance_config(
+            base_dir, "legacy_run_queue_enabled_run_blocking.yaml"
+        )
+        with instance_for_test(overrides=instance_config) as instance:
+            concurrency_config = instance.get_concurrency_config()
+            run_queue_config = concurrency_config.run_queue_config
+            assert run_queue_config
+            assert run_queue_config.max_concurrent_runs == 5
+            assert run_queue_config.tag_concurrency_limits == [
+                {
+                    "key": "dagster/solid_selection",
+                    "limit": 2,
+                }
+            ]
+            assert run_queue_config.max_user_code_failure_retries == 3
+            assert run_queue_config.user_code_failure_retry_delay == 10
+            assert run_queue_config.op_concurrency_slot_buffer == 1
+            assert run_queue_config.should_block_op_concurrency_limited_runs
+            assert concurrency_config.pool_config.pool_granularity == PoolGranularity.OP
```

Comment:
seems like the answer to my question above is 'yes' given that there is still max_concurrent runs but the granularity is OP